### ISSUE TITLE
Starting server with env_path argument given as relative path

### ISSUE THREAD

**Issue by ohwi** [fossasia/visdom#568]
Hi.

I wanted to save environment json files to somewhere I want, so I executed following code in that folder:

```{bash}
python -m visdom.server -env_path='.'
```

However, I got following assertion error:

```
Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 51, in ensure_dir_exists
    os.makedirs(os.path.dirname(path))
  File "/anaconda3/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
FileNotFoundError: [Errno 2] No such file or directory: ''

Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 1184, in <module>
    download_scripts_and_run()
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 1180, in download_scripts_and_run
    main()
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 1175, in main
    print_func=print_func, user_credential=user_credential)
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 1112, in start_server
    user_credential=user_credential)
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 131, in __init__
    ensure_dir_exists(env_path)
  File "/anaconda3/lib/python3.7/site-packages/visdom/server.py", line 53, in ensure_dir_exists
    assert e1.errno == 17  # errno.EEXIST
AssertionError
```

I wonder if this assertion is intended.

If not, I think printing warning or getting dirname with abspath may need.

**Comment by JackUrb** [comment]
Hi @ohwi, This should be fixable by using abspath - using a full pathname should work as expected at the moment. If you want to add that to the implementation, feel free to open a PR!

**Comment by JackUrb** [comment]
Resolved in #839
